client/src/components/Ticketing/ticketingmanager/showings/ShowingUpdated/TicketTypeUpdateTable.tsx

```diff
@@ -0,0 +1,190 @@
+import {Field, useField} from 'formik';
+import React, {useState} from 'react';
+import {TicketTypeSelect} from './TicketTypeSelect';
+import {InputControl} from './InputControl';
+import {IconButton} from '@mui/material';
+import {useEvent} from './EventProvider';
+import {getTicketTypePrice} from './ShowingUtils';
+
+interface TicketTypeTableProps {
+  arrayHelpers;
+  eventInstanceID: number;
+}
+
+export const TicketTypeUpdateTable = (props: TicketTypeTableProps) => {
+  const {arrayHelpers, eventInstanceID} = props;
+  const {ticketTypes, showPopUp} = useEvent();
+  const [field] = useField('instanceTicketTypes');
+  const [availableTypes, setAvailableTypes] = useState(
+    ticketTypes
+      .filter(
+        (type) =>
+          !field.value.find((value) => value.typeID == type.id) && type.id != 1,
+      )
+      .map((value) => value.id),
+  );
+
+  return (
+    <div
+      className={
+        'overflow-y-auto overflow-x-auto col-span-12 min-[1350px]:col-span-7 ' +
+        'shadow-xl border-4 border-gray-300 mx-auto rounded-xl bg-white w-[100%] min-h-[100px]'
```

Comment:
I have a couple more minor style comments, but do you think you could remove the gray border from this component?